autofixture/generators.py

```diff
@@ -664,3 +665,15 @@ class UUIDGenerator(Generator):
 
     def generate(self):
         return uuid.uuid4()
+
+
+# Geo
+class PointFieldGenerator(Generator):
+
+    latitude = 19.99
+    longitude = 73.78
```

Comment:
How did you choose these values? They seem arbitrary to me. Would it make sense to have them here set to `latitude=0` and `longitude=0`?